### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by TimothyChilvers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+- Do not include build folders or IDE specific files.
+
+# Endpoints
+- The procedures list will be available at `https://staging.touchsurgery.com/tech-test/procedures` 
+- Procedure details are available at `https://staging.touchsurgery.com/tech-test/procedures/$PROCEDURE_ID`
+
+For example `https://staging.touchsurgery.com/tech-test/procedures/procedure-TSC_CemCup`
+
+# Technologies to use:
+You will be joining an existing team with existing technology. We are interested in guaging how well you use those technologies, so if we do not see them addressed here we will discuss them in the interviews. These technologies include, but are not limited to:
```
Comment (MiguelSesma) [inline README.md:25]: Typo: gauging instead guaging?

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
-# android-tech-test
\ No newline at end of file
+# android-tech-test
+
+We would like you to spend a few hours on the following technical task:
+
+Create an app with two screens:
+
+# Procedure Page:
+This should be the home page of the app. It should have a list of procedures cards/cells, each with a thumbnail image and a name. This information can be acquired from the procedures endpoint (/procedures). Clicking on one of the procedure cards/cells should take the user to the next page.
```
Comment (ibracero) [inline README.md:8]: the procedures endpoint (/procedures) - see endpoints section below?

**Review by ibracero — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TimothyChilvers) [reply README.md:25]: Copy and paste error >_>

**Review by MiguelSesma — APPROVED** [review]

**Review by MichaelRMedtronic — APPROVED** [review]
Looks good. It fixes the issues with the previous test description. 

I'm a little concerned that expecting Room, Dagger/Hilt and UI tests makes the test too long. Each could add an hour of work.

**Pull request merged**